<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>classical_electrophysiology_widget</class>
 <widget class="QWidget" name="classical_electrophysiology_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="classical_electrophysiology">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="3" column="2">
       <widget class="QPushButton" name="record_trace_button">
        <property name="text">
         <string>Record Trace</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="classical_electrophysiology_label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Classical electrophysiology</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QRadioButton" name="record_electrophysiological_trace_radioButton">
        <property name="text">
         <string>Record electrophysiological trace ?</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="display_trace_button">
        <property name="text">
         <string>Display Trace</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="stimulation_button">
        <property name="text">
         <string>Stimulation</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="close_graph_button">
        <property name="text">
         <string>Close Graph</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
